configuration/rules-dsl.md

```diff
@@ -206,6 +206,40 @@ For example, one z-wave device can be "zwave:device:c5155aa4:node14".
 > Note: You need to use quotes around `thingUID` if it contains special characters such as ':'.
 > The parser will not recognize the whole text correctly without quotes.
 
+{: #channel-based-triggers}
+### Channel-based Triggers
+
+You can listen for triggers from specific *trigger channels* that are provided by some addons.
+You can decide whether you want to catch only a specific trigger or any.
+Here is the syntax for all these cases (parts in square brackets are optional):
+
+```java
+Channel "<triggerChannel>" triggered [<triggerEvent>]
```

Comment:
This is already the generic example.
You can listen for any trigger of that channel or for a special one.
(According to my comment above about that "any".)

Reply:
So the way this works, you add a "Channel" statement for any `triggerChannel` you want to catch - correct?  If so, then I am suggesting the example look like this...
```java
Channel "<triggerChannel1>" triggered [<triggerEvent1>]
Channel "<triggerChannel2>" triggered [<triggerEvent2>]
```
Where `triggerChannel1` and `triggerChannel2` are different triggers that you can use to trigger `triggerEvent1` and `triggerEvent2`, respectively.

This would make it completely clear that it is one triggerChannel per line and not that you can somehow add multiple events in one line, as long as they are all from the same binding.  Or...

Just forget it and leave it as-is.  Maybe it is completely obvious.  I am happy either way you decide to go.